apps/ledger-live-desktop/src/renderer/components/AccountSubHeader/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import Box, { Card } from "~/renderer/components/Box";
+import Text from "~/renderer/components/Text";
+import { Trans, useTranslation } from "react-i18next";
+import Button from "~/renderer/components/Button";
+import AccountSubHeaderDrawer from "./AccountSubHeaderDrawer";
+import ChevronRight from "~/renderer/icons/ChevronRight";
+import InfoCircle from "~/renderer/icons/InfoCircle";
+
+const CardContent = styled(Box)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  flex: 1;
+`;
+
+const CardHeaderContainer = styled(Box)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+`;
+
+const CardHeader = styled(Text)`
+  font-weight: 600;
+  font-size: 12px;
+  margin-left: 8px;
+`;
+
+const CustomButton = styled(Button)`
+  border: none;
+  padding-right: 14px;
+`;
+
+export default function AccountSubHeader({ family, team }) {
```

Comment:
For some reason I cannot add type Props = { family: string, team: string } and use Props type here... build fails